fury/window.py

```diff
@@ -32,6 +33,21 @@ class Scene(Renderer):
     but also it provides access to all the functionality
     available in ``vtkRenderer`` if necessary.
     """
+    def __init__(self, background=(0, 0, 0), skybox_tex=None,
+                 render_skybox=False):
```

Comment:
One variable I think is enough. You do not need ``skybox_tex`` and ``render_skybox``.
I would use ``skybox = None`` only.

Reply:
And make some additional functions to remove or update the skybox as needed.

Reply:
The second parameter is to control when to show or not the skybox. I could take that part out of the `__init__` function and add function like `scene.show_skybox(params)`

Reply:
Yes, that is better. If the user add the skybox textures in Scene then the skybox should appear by default but if people want to hide it then that should be possible too. 
In the new function please do not use scene.show_skybox but just say scene.skybox.on() or off() or scene.skybox(True or False).

Reply:
Or even better scene.skybox = True or False and allow for another parameter ``scene.skybox_filename = '...\..png'``.

Reply:
Ok, I will adjust accordingly.

Reply:
So that people can even change the skybox during runtime. These changes please do them in another PR. But do use one parameter in Scene init in this PR. Thank you @guaje.

Reply:
Done. I have removed the `render_skybox` param and moved its functionality to the `show_skybox` function. Additionally, I have added the `hide_skybox` function.